draft-ietf-anima-brski-prm.md

```diff
@@ -520,127 +540,146 @@ In {{!RFC8995}}, pledges instead need to continuously request enrollment from a
 The benefits of BRSKI-PRM can be achieved even without the operational complexity of standalone Registrar-Agents by integrating the necessary functionality of the Registrar-Agent as a module into the domain registrar as shown in {{uc4figure}} so that it can support the BRSKI-PRM communications to the pledge.
 
 
-## Agent-Proximity Assertion {#agt_prx}
 
-"Agent-proximity" is a statement in the PVR and in the voucher, that the registrar certificate was provided via the Registrar-Agent as defined in {{exchanges_uc2}} and not directly to the pledge.
-Agent-proximity is therefore a different assertion than "proximity", which is defined in {{Section 4 of RFC8366}}.
-Agent-proximity is defined as additional assertion type in {{I-D.ietf-anima-rfc8366bis}}.
-This assertion can be verified by the registrar and also by the MASA during the voucher-request processing.
+## Agent Proximity Assertion {#agt_prx}
+
+"Agent proximity" is a statement in the PVR and in Voucher that the registrar LDevID certificate was provided via the Registrar-Agent as defined in {{exchanges}} and not directly to the pledge.
+It is therefore a different assertion than "network proximity", which is defined in {{Section 3 of !RFC8995}}.
+Hence, {{I-D.ietf-anima-rfc8366bis}} defines the additional assertion type `agent-proximity`.
+This assertion type can be verified by the registrar and MASA during BRSKI-PRM voucher-request processing.
 
-In BRSKI, the pledge verifies POP of the registrar via the TLS handshake and pins that public key as the "proximity-registrar-cert" into the voucher request.
+In BRSKI, the pledge verifies POP of the registrar via the TLS handshake and pins that public key as the `proximity-registrar-cert` into the voucher request.
 This allows the MASA to verify the proximity of the pledge and registrar, facilitating a decision to assign the pledge to that domain owner.
 In BRSKI, the TLS connection is considered provisional until the pledge receives the voucher.
 
-In contrast, in BRSKI-PRM, the pledge has no direct connection to the registrar and MUST accept the registrar certificate provisionally until it receives the voucher as described in {{voucher}}.
-In a similar fashion, the pledge MUST accept the Registrar-Agent EE certificate provisionally.
+In contrast, in BRSKI-PRM, the pledge has no direct connection to the registrar and MUST accept the registrar LDevID certificate provisionally until it receives the voucher as described in {{voucher}}.
+In a similar fashion, the pledge MUST accept the Registrar-Agent LDevID certificate provisionally. TODO(is this HTTPS-related? otherwise why accept it at all?)
```

Comment:
This is to resemble the provisional accept state from BRSKI.  It intended to create the state for the registrar-certificate and to ensure the one in the agent-provided- registrar-certificate correlates to the one in the voucher.

Reply:
Clarified in chat that there is no acceptance of the RegAgt EE certificate (just mirroring of the agent-signed data).